analyses/molecular-subtyping-LGAT/04-LGAT-compile-subtypes.Rmd

```diff
@@ -0,0 +1,290 @@
+---
+title: "R Notebook"
+output: html_notebook
+---
+
+In this notebook, we will be compiling the subtype annotation for LGAT samples. Subtypes are described in [#790](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/790) which were gathered through the following scripts : 
+
+`01-subset-files-for-LGAT.Rmd` gathers subtype associated SNV annotation 
+`02-subset-fusion-files-LGAT.Rmd` gathers subttype associated Fusion annotation
+`03-subset-cnv-files-LGAT.Rmd` gathers subtype associated CNV annotation
+
+along with the above molecular data based annotation in this notebook we will include NF-1 cancer_predispositions from histology file as a criteria for LGG, NF1 as well
+
+### Setup
+```{r}
+library("tidyverse")
+```
+
+### Input
+```{r}
+# Look for git root folder
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# clinical file
+lgat_manifest <- read_tsv(file.path(root_dir, 
+                               "data",
+                               "pbta-histologies.tsv"), 
+                     guess_max = 10000) %>%
+  filter( broad_histology == "Low-grade astrocytic tumor") %>%
```

Comment:
Should this be using the same inclusion/exclusion criteria as what is used elsewhere and described here: https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/0ed49b68a62462d2c96495ffea1bdadd5b4ff835/analyses/molecular-subtyping-LGAT#inclusionexclusion-criteria ?

Reply:
Yes I think so, I can add the inclusion/exclusion from the yaml file used before

Reply:
hmm using the inclusion/exclusion from the yaml file removes "7316-1106" which was updated to Low grade from path review. 

https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/f85831fe7e405b1c6b6c43274cabcc796b7bd3b1/analyses/molecular-subtyping-pathology/03-incorporate-pathology-feedback.Rmd#L199-L224

Reply:
I think it should be reverted back to broad_histology filter, thoughts?

Reply:
My understanding is that `broad_histology` depends on `integrated_diagnosis` (or would depend on in the future), which depends on `molecular_subtype` - I'm trying to avoid circularity.

Reply:
I actual forgot to use `pbta-histologies-base.tsv` since it is a moleuclar-subtyping module, I'am updating that.

Reply:
I skimmed the HGG module and the earlier notebook in this module - it seems like `pbta-histologies-base.tsv` and using the JSON file is the way to go. Is that correct?

Reply:
correct!